src/main/kotlin/com/terraformation/backend/funder/model/FundingEntityModel.kt

```diff
@@ -12,16 +12,20 @@ data class FundingEntityModel(
     val name: String,
     val createdTime: Instant,
     val modifiedTime: Instant,
-    val projects: List<ProjectId> = emptyList(),
+    var projects: List<ProjectId> = emptyList(),
```

Comment:
I think it's highly preferable to keep model classes immutable. Copy overhead isn't a big deal for our application.

Reply:
Ah yes, I had only modified this to make the projectsField stuff work, but your second comment fixes that. Updated!